.github/actions/spelling/expect.txt

```diff
@@ -62,8 +64,10 @@ curated
 CYRL
 deigh
 deleteifnotneeded
+delimeter
```

Comment:
```suggestion
```

Reply:
removed

---
In reply to: [612012989](https://github.com/microsoft/winget-cli/pull/861#discussion_r612012989) [](ancestors = 612012989)